cfgov/paying_for_college/jinja2/paying-for-college/college-costs.html

```diff
@@ -48,7 +48,16 @@
                             {%- include 'college-cost-blocks/16-save-and-finish.html' with context -%}
                             <div class="college-costs_tool-section_buttons block block__mid">
                                 <button class="a-btn college-costs_buttons__next btn__next-step">Next step</button>
-                                <button class="a-btn a-btn__link btn__save">Save and finish later</button>
+                                <button class="a-btn a-btn__link btn__save" data-app-button="save-and-finish-later">Save and finish later</button>
+                            </div>
+                            <div class="college-costs_save-for-later block block__mid" data-state-based-visibility="save-for-later">
+                                <p>Copy and save this link so that you can come back and review your plan.</p>
+                                <div class="m-form-field">
+                                    <p>
```

Comment:
Should this get emphasis -- maybe an `.h4` class?

Reply:
As above - I don't have a strong opinion here. :) I was figuring someone would come along and critique the styles so we could just discuss it from there. ;-)

I'll bring it up on our Internal Dialogue Platform of Choice.

Reply:
See above - I thought I emphasized this one, not the last one.